app/presenters/development_presenter.rb

```diff
@@ -3,12 +3,16 @@ class DevelopmentPresenter < Burgundy::Item
   # Relationships
 
   # Everyone who has provided data for this development
-  def contributors
+  def preview_contributors
     # TODO Wrap contributors in ContributorPresenter
-    %w( mark lena lindsay molly eve ).shuffle
+    ContributorPresenter.wrap item.contributors.first(5).shuffle
     # item.contributors.first(5) # TODO Optimize query instead of getting all contributors
   end
 
+  def contributors
+    ContributorPresenter.wrap item.contributors.sort_by(&:last_name)
```

Comment:
Diggin' this. I think the sort_by is smart: this gets around the issue where `#order(:last_name)` would be incompatible with adding in the contributor, and keeps presentation logic (the ordering) in the presenter.